### ISSUE TITLE
Inline plotting

### ISSUE THREAD

**Issue by MLKaufman** [quarto-dev/quarto#28]

**Comment by peekxc** [comment]
Though not quite the same feature, looks like you can now embed jupyter cell outputs in the newest prerelease (https://quarto.org/docs/prerelease/1.3/embed.html)

**Comment by emfeltham** [comment]
I agree that that feature is nice, but I think that it would be really useful to have inline plotting. Generally, if I want that now in vscode, I have to use Jupyter notebooks. I much, much prefer working with rmd-style documents; though, I've been using Jupyter notebooks lately for two reasons (1) inline plotting makes it much easier to see what's going on when I am working on a longer document, and (2) the outputs stay embedded in the document, and I don't have to re-run all of the cells to generate a document when I add new cells or modify some subset of them. These are weighted against what I find to be a more awkward experience, especially for longer prose sections.

**Comment by juliasilge** [comment]
We have this feature now in Positron:
https://positron.posit.co/quarto-inline-output.html
